### PR TITLE
fix: Resend code & change mail

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: app/script/auth/page/Verify.js [hunk 1]
```diff
@@ -48,16 +56,16 @@ const Verify = ({account, authError, history, ...connected}) => {
...
+            {account.email}
           </Text>
-          <CodeInput data-uie-name="enter-code" autoFocus style={{marginTop: 10}} onCompleteCode={createAccount} />
+          <CodeInput autoFocus style={{marginTop: 10}} onCompleteCode={createAccount} data-uie-name="enter-code" />
           <ErrorMessage>{authError}</ErrorMessage>
         </div>
         <div>
-          <Link data-uie-name="do-resend-code" href="#">
+          <Link onClick={resendCode} data-uie-name="do-resend-code">
             {'RESEND CODE'}
```
Comment (gregor) [inline app/script/auth/page/Verify.js:66]: Can we transform the link text to uppercase and stop screaming here?

File: app/script/auth/page/Verify.js [hunk 2]
```diff
@@ -48,16 +56,16 @@ const Verify = ({account, authError, history, ...connected}) => {
...
           </Link>
-          <Link data-uie-name="go-change-email" href="#" style={{marginLeft: 35}}>
+          <Link to={ROUTE.CREATE_ACCOUNT} component={RRLink} style={{marginLeft: 35}} data-uie-name="go-change-email">
             {'CHANGE EMAIL'}
```
Comment (gregor) [inline app/script/auth/page/Verify.js:69]: Can we transform the link text to uppercase and stop screaming here?

File: app/script/auth/page/Verify.js (hunk 1 above)
Comment (Yserz) [reply app/script/auth/page/Verify.js:66]: @AndyLnd is on it already

File: app/script/auth/page/Verify.js (hunk 2 above)
Comment (Yserz) [reply app/script/auth/page/Verify.js:69]: @AndyLnd is on it already

**Pull request merged**